healthyregions/chicago-environment-explorer
# feat: added Lead Poisoning Rate (Ages 1-5) as a new variable

## Problem
We have a new variable submission for "Lead Poisoning Rate (Ages 1-5)" that has been submitted to the platform

## Approach
* feat: added the new variable to the ChiVes map

![Screenshot 2024-08-08 at 11 59 45 AM](https://github.com/user-attachments/assets/36c8889e-53a9-4a98-ab7b-0cb1fa5bfec7)

## How to Test
1. Navigate to https://deploy-preview-186--chicago-env-explorer.netlify.app/map
2. From the Variables dropdown, select "Lead Poisoning Rate (Ages 1-5)"
    * You should see the new variable is visualized on the map
    * You should see that the colorScale matches other Health-based variables that we offer
